docs/kura-wires/script-components/nashorn-script-filter.md

```diff
@@ -95,4 +96,49 @@ record['myprop'] = newDoubleValue(123.456)
 
 // add the wire record to the output envelope
 output.add(record)
-```
\ No newline at end of file
+```
+
+#### Utility functions
```

Comment:
We should insert this section also for the GraalVM based component. I would propose two ways for integrating this documentation piece:

1. Put the commonalities into the "Introduction to the Script Components" page and display in the Nashron- and GraalVM pages only the things that differ (newWireRecord(void) utility function, "Received envelope" and "Creating and emitting wire records"). 
2. Get rid of this section (not really informative) and complete the examples in this page with all the wrapper types like for the GraalVM section, with some JS comments if needed. This should be enough.